cloudflare/src/index.js

```diff
@@ -107,6 +107,41 @@ router
     }
   })
 
+  // TEMPORARY DEBUG ENDPOINT - Remove after troubleshooting (before auth)
+  .get('/api/debug/permissions', async (request, env) => {
```

Comment:
instead of debug endpoints you could inspect the users permissions via network calls

you can see the users permission there without code

<img width="762" height="580" alt="image" src="https://github.com/user-attachments/assets/d7996e38-f4f9-4533-94f3-35e9e3cfc284" />